getting-started/android-frontend-with-appengine-backend/Frontend/build.gradle

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+buildscript {
+    ext.kotlin_version = '1.2.30'
```

Comment:
Consider using the latest Kotlin version - https://blog.jetbrains.com/kotlin/category/releases/, See https://bintray.com/bintray/jcenter/org.jetbrains.kotlin%3Akotlin-gradle-plugin. `1.2.61` is the latest.